<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>printPage</class>
 <widget class="QWidget" name="printPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>313</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Печать</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="lblEnableFastPrint">
     <property name="text">
      <string>Разрешить печать без показа диалога настройки принтера</string>
     </property>
     <property name="buddy">
      <cstring>chkEnableFastPrint</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QCheckBox" name="chkEnableFastPrint">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer_9">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblShowPageSetup">
     <property name="text">
      <string>Отображать окно настроек параметров страницы</string>
     </property>
     <property name="buddy">
      <cstring>chkShowPageSetup</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="chkShowPageSetup">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer_11">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>49</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblLabelPrinter">
     <property name="text">
      <string>Принтер для визиток пациентов и наклеек</string>
     </property>
     <property name="buddy">
      <cstring>cmbLabelPrinter</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QComboBox" name="cmbLabelPrinter"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblPrintBlackWhite">
     <property name="text">
      <string>Отключать цвет при выводе на печать</string>
     </property>
     <property name="buddy">
      <cstring>chkPrintBlackWhite</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="chkPrintBlackWhite">
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <spacer name="horizontalSpacer_7">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lblEnablePreview">
     <property name="text">
      <string>Разрешить печать с предпросмотром</string>
     </property>
     <property name="buddy">
      <cstring>chkEnablePreview</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QCheckBox" name="chkEnablePreview">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <spacer name="horizontalSpacer_10">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>90</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lblTemplateEdit">
     <property name="text">
      <string>Режим отладки шаблонов печати</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QCheckBox" name="chkTemplateEdit">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>chkEnableFastPrint</tabstop>
  <tabstop>cmbLabelPrinter</tabstop>
  <tabstop>chkPrintBlackWhite</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
